examples/singlemodel/object_detection_annotate_video_file.ipynb

```diff
@@ -84,55 +93,49 @@
    },
    "outputs": [],
    "source": [
-    "import degirum as dg, degirum_tools\n",
-    "from pathlib import Path\n",
-    "import IPython.display\n",
+    "import degirum as dg\n",
+    "import degirum_tools\n",
     "\n",
-    "# configure for Google Colab\n",
-    "degirum_tools.configure_colab()\n",
-    "# connect to AI inference engine getting token from env.ini file\n",
+    "# Connect to AI inference engine\n",
     "zoo = dg.connect(hw_location, model_zoo_url, degirum_tools.get_token())\n",
-    "# load object detection model\n",
+    "# Load object detection model\n",
     "model = zoo.load_model(\n",
     "    model_name, overlay_show_probabilities=True, overlay_line_width=1\n",
     ")\n",
-    "orig_path = Path(input_filename)\n",
-    "ann_path = \"../../workarea/\" + orig_path.stem + \"_annotated\" + orig_path.suffix\n",
-    "degirum_tools.annotate_video(model, orig_path, ann_path)"
+    "# Annotate the video using the loaded model\n",
+    "degirum_tools.annotate_video(model, video_source, ann_path)"
```

Comment:
This really isn't an example. We should either make a subfolder for demos, or provide a link to the degirum_tools source where the actual code is.

Reply:
@AlexABolotov Not sure I understand your comment. This is an example that shows how to annotate a video file using the `annotate_video` functionality of `degirum_tools`

Reply:
The example doesn't teach anything about annotating videos with PySDK (which is what it says in the header of it. Is degirum tools part of PySDK? Not trying to be pedantic, but that line set an expectation for me that was not borne out by the actual code further down). It rather showcases that we have an annotation function in our nifty support library. That's great if someone is solving a problem that this premade function happens to solve to their specification. To me, that's a very narrow use case. What if someone already has a video annotator, and wants to swap the AI call to PySDK? How do they handle the result? How do they do the actual drawing? All of that is hidden. So the pedagogical value is very low.
I think we need to be clearer both during the development of the "samples" and in their presentation what the goal of the sample actually is, as well as the place and purpose of the degirum tools library. To me "examples" are educational, and should showcase _how_ to use PySDK, both in a basic sense and as part of larger solution. On the other hand, "demos" are just turnkey showcases, easy for us to deploy somewhere to show our stack/hardware to people uninterested in how it all works. If we actually see degirum tools as some sort of app-building-block library that can be useful to people besides us, we can also have "example apps", that showcase degirum tools.
When I looked at this file, its location in "examples/singlemodel" and the intro text led me to believe that it would be an "example" as described above. I think degirum tools here just nullifies the value of the code as an "example". At the same time, it's too basic to really be an "example app".

Reply:
@AlexABolotov understood. However, we cannot really do anything about this in this release. We will revisit this in next round of refactoring.